src/core/Akka.API.Tests/verify/CoreAPISpec.ApproveCore.DotNet.verified.txt

```diff
@@ -269,6 +269,7 @@ namespace Akka.Actor
         public bool Equals(Akka.Actor.IActorRef other) { }
         public override int GetHashCode() { }
         public void Tell(object message, Akka.Actor.IActorRef sender) { }
+        protected virtual object TellInterceptor(object message) { }
```

Comment:
We should probably solve this issue internally inside the scheduler, rather than through new public API methods on the actor ref implementations.

Reply:
done